actors/builtin/multisig/multisig_test.go

```diff
@@ -206,7 +206,7 @@ func TestVesting(t *testing.T) {
 		rt.SetCaller(anne, builtin.AccountActorCodeID)
 		rt.ExpectValidateCallerType(builtin.AccountActorCodeID, builtin.MultisigActorCodeID)
 		rt.ExpectAbort(exitcode.ErrInsufficientFunds, func() {
-			actor.propose(rt, darlene, abi.NewTokenAmount(100), builtin.MethodSend, fakeParams, nil)
+			_ = actor.propose(rt, darlene, abi.NewTokenAmount(100), builtin.MethodSend, fakeParams, nil)
```

Comment:
FYI @anorth It felt confusing to use `proposeOK` and `approveOK` in `ExpectAbort` clauses that we know should fail so to get around linter's errcheck on the exitcode I explicitly throw away the exitcode value within these clauses.  This seems better since we're communicating that we don't care what value we get, any is wrong.